modules/compliance-scheduling-pods-with-resource-requests.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+:_content-type: CONCEPT
+[id="compliance-scheduling-pods-with-resource-requests"]
+= Scheduling Pods with resource requests
+
+When a Pod is created, the scheduler selects a Node for the Pod to run on. Each node has a maximum capacity for each resource type in the amount of CPU and memory it can provide for the Pods. The scheduler ensures that the sum of the resource requests of the scheduled containers is less than the capacity nodes for each resource type.
+
+Although memory or CPU resource usage on nodes is very low, the scheduler might still refuse to place a Pod on a node if the capacity check fails to protect against a resource shortage on a node.
+
+For each container, you can specify resource limits and requests, including the following:
```

Comment:
Is this the entire list? If so, "For each container, you can specify the following resource limits and request:"

Either way, you need a noun after "following."